client/stylesheets/_toCat.scss

```diff
@@ -4832,16 +4834,26 @@ On click the images will be displayed at normal size to complete the effect
 	position:absolute;
 	left:0;
 	right:0;
+	border: 2px solid #f39d00;
+	margin-top: -2px;
 }
 
 .optionsClick ul .clicker {
 	position:relative;
-	background:#2284B5;
+	background:orange;
 	color:#FFF;
+	text-align: center;
+	padding: 20px 0;
+	border-radius: 4px;
+}
+
+.optionsClick ul .clicker i {
+	color: #fff !important;
+	margin: 0 0 -3px 8px;
 }
 
 .optionsClick ul .clicker:hover,.click-nav ul .active {
-	background:#196F9A;
+	background:#F39D00;
```

Comment:
Color literals like `#F39D00` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F39D00` should be written as `#f39d00`<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space